### PR TITLE
Setting transaction fee when starting relayer

### PR DESCRIPTION
**Pull request by marcelomorgado**

### CODE REVIEW

**Comment by nventuro** [comment]
Great catch @marcelomorgado! 

Could you also update https://github.com/OpenZeppelin/openzeppelin-gsn-helpers/blob/master/oz-gsn-run-relayer.js so that `--fee` option is available when using the Helpers from the command line? Thanks a lot!

**Comment by marcelomorgado** [comment]
> Great catch @marcelomorgado!
> 
> Could you also update https://github.com/OpenZeppelin/openzeppelin-gsn-helpers/blob/master/oz-gsn-run-relayer.js so that `--fee` option is available when using the Helpers from the command line? Thanks a lot!

Sure, done. You are welcome!

File: oz-gsn-run-relayer.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ program
   )
   .option('--no-register', 'skip registration of the relayer process')
   .option('--no-devMode', 'turns off dev mode in relayer')
+  .option('--fee', "relay's per transaction fee")
```
Comment (nventuro) [inline oz-gsn-run-relayer.js]:
```suggestion
  .option('--fee <percentage>', "relayer's transaction fee (defaults to 70)")
```

Could you also add a note to the documentation of [runRelayer](https://github.com/OpenZeppelin/openzeppelin-gsn-helpers/blob/master/docs/modules/ROOT/pages/api.adoc#runrelayer) so this shows up on the [docs site](https://docs.openzeppelin.com/gsn-helpers/0.2/api#runrelayer)? Thanks!

Comment (marcelomorgado) [reply oz-gsn-run-relayer.js]: Sure @nventuro, done.

**Pull request merged**